### PR TITLE
adding examples for the new windows_user_privilege and windows_securi…

### PR DESCRIPTION
**Pull request by sam1el**
I have added examples for the new windows_user_privilege and windows_security_policy

Signed-off-by: Jeff Brimager <jbrimager@chef.io>

### Description

The current examples for these 2 resource are a little too generic. With the complexities of what these resources manage, I feel like the examples should be a bit more clear. Mostly for newer users who may not be familiar with managing windows who are working from a security document.  

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [x] Spell Check
- [x] Local build
- [x] Examine the local build
- [x] All tests pass

### CODE REVIEW

**Comment by sam1el** [comment]
Need to edit 1 or 2 things I missed

File: content/resources/windows_user_privilege/_index.md [hunk 1]
```diff
@@ -59,5 +59,44 @@ properties_list:
...
+
+  ```ruby
+  windows_user_privilege 'Remote interactive logon' do
+    privilege      'SeDenyRemoteInteractiveLogonRight'
+    users          ['Builtin\Guests', 'NT AUTHORITY\Local Account']
+    action         :add
+  end
+  ```
+
+  **Enforce only Builtin Guests and Adminiistrator Gruops allowed for SeCreatePageFile Privilege**:
```
Comment (IanMadd) [inline content/resources/windows_user_privilege/_index.md]:
```suggestion
  **Provide only the Builtin Guests and Administrator Groups with the SeCreatePageFile Privilege**:
```

File: content/resources/windows_user_privilege/_index.md [hunk 2]
```diff
@@ -59,5 +59,44 @@ properties_list:
   description_list:
   - markdown: An optional property to set the privilege for given users. Use only
       with set action.
-examples: 
----
\ No newline at end of file
+examples: |
+  **Enforce only Builtin Administrators Group and Authenticated Users allowed for the SeNetworkLogonRight Privilege**:
```
Comment (IanMadd) [inline content/resources/windows_user_privilege/_index.md]:
```suggestion
  **Set the SeNetworkLogonRight Privilege for the Builtin Administrators Group and Authenticated Users**:
```

File: content/resources/windows_user_privilege/_index.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ actions_list:
   :remove:
     markdown: Remove a user privilege
   :set:
-    markdown: Set the complete set of user privileges
+    markdown: This will remove any user not listed in the `users` field and only allow what is listed what was provided to remain.
```
Comment (IanMadd) [inline content/resources/windows_user_privilege/_index.md]:
```suggestion
    markdown: Set the privileges that are listed in the `privilege` property for only the users listed in the `users` property.
```

File: content/resources/windows_user_privilege/_index.md [hunk 4]
```diff
@@ -59,5 +59,44 @@ properties_list:
...
+
+  ```ruby
+  windows_user_privilege 'Create Pagefile' do
+    privilege      'SeCreatePagefilePrivilege'
+    users          ['BUILTIN\Guests', 'BUILTIN\Administrators']
+    action         :set
+  end
+  ```
+
+  **Remove Builtin Guests from SeCreatePageFile Privilege**:
```
Comment (IanMadd) [inline content/resources/windows_user_privilege/_index.md]:
```suggestion
  **Remove the SeCreatePageFile Privilege from the Builtin Guests Group**:
```

File: content/resources/windows_user_privilege/_index.md [hunk 5]
```diff
@@ -59,5 +59,44 @@ properties_list:
...
+
+  ```ruby
+  windows_user_privilege 'Netowrk Logon Rights' do
+    privilege      'SeNetworkLogonRight'
+    users          ['BUILTIN\Administrators', 'NT AUTHORITY\Authenticated Users']
+    action         :set
+  end
+  ```
+
+  **Add Builtin Guests and Local Accounts to the SeDenyRemoteInteractiveLogonRight Privilege**:
```
Comment (IanMadd) [inline content/resources/windows_user_privilege/_index.md]:
```suggestion
  **Add the SeDenyRemoteInteractiveLogonRight Privilege to the Builtin Guests and Local Accounts User Groups**:
```

**Review by IanMadd — COMMENTED** [review]
Suggestions for clarity. Let me know what you think @sam1el

**Comment by sam1el** [comment]
@IanMadd These all make perfect sense. I went ahead and made the changes and they have been pushed.

**Review by IanMadd — APPROVED** [review]

**Pull request merged**